eng/versioning/version_client.txt

```diff
@@ -17,8 +17,8 @@ com.azure:azure-core-serializer-json-jackson;1.0.0-beta.1;1.0.0-beta.1
 com.azure:azure-core-test;1.2.0;1.3.0-beta.1
 com.azure:azure-core-tracing-opentelemetry;1.0.0-beta.4;1.0.0-beta.5
 com.azure:azure-cosmos;4.0.1-beta.2;4.0.1-beta.3
-com.azure:azure-cosmos-examples;4.0.1-beta.1;4.0.1-beta.1
-com.azure:azure-cosmos-benchmark;4.0.1-beta.1;4.0.1-beta.1
+com.azure:azure-cosmos-examples;4.0.1-beta.1;4.0.1-beta.3
```

Comment:
@kushagraThapar 
com.azure:azure-cosmos-examples and azure-cosmos-benchmark are in the process is being removed as libraries. Further, they're intentionally not part of the Artifacts cosmos' [ci.yml](https://github.com/Azure/azure-sdk-for-java/blob/master/sdk/cosmos/ci.yml#L52) which makes them incapable of being released (this was intentional as these are never supposed to be released). There is no version change necessary here and it should be reverted.

Reply:
@JimSuplizio - yes this is unintentional, should be reverted. 
@David-Noble-at-work - can we please revert the changes related to pom.xml ?

Reply:
@kushagraThapar @JimSuplizio yep